src/expression/comparison/index.ts

```diff
@@ -25,8 +30,10 @@ export abstract class Comparison implements Evaluable {
     protected readonly operator: string,
     protected readonly operatorSymbol: symbol,
     protected readonly left: Operand,
-    protected readonly right: Operand) {
-  }
+    protected readonly right: Operand
+  ) {}
+
+  public strict = true;
```

Comment:
Please add the comment, as suggested (you can use your words). You should really describe the meaning, and default behavior. (as open source, we should be really clear)

Reply:
d448793